namerd/docs/interface.md

```diff
@@ -40,9 +40,243 @@ addrCacheInactive | `100` | The size of the address inactive cache.
 
 kind: `io.l5d.httpController`
 
-A read-write HTTP interface to the [storage](#storage).
+The HTTP controller provides APIs for reading and writing dtabs, as well as for
+viewing how names are resolved.
 
 Key | Default Value | Description
 --- | ------------- | -----------
 ip | loopback | The local IP address on which to serve the namer interface.
 port | `4180` | The port number on which to serve the namer interface.
+
+### GET /api/1/dtabs
+
+> Sample response
+
+```
+["default"]
+```
+
+Returns a list of all dtab namespaces.
+
+Content-types: application/json
+
+### GET /api/1/dtabs/&lt;namespace&gt;
+
+> Sample response
+
+```
+[{"prefix":"/http/1.1/GET","dst":"/#/io.l5d.fs"}]
+```
+
+Returns the requested dtab.  The dtab version is returned in the Etag response
+header.  Returns status code 404 if the dtab namespace does not exist.
+
+Parameter | Type | Description
+--------- | ---- | -----------
+namespace | path | The dtab namespace to retrieve.
+watch | uri | If true, updates are returned as a streaming response.
```

Comment:
I think I'll hold off for now, just because it either make things inconsistent if I only apply it to the new tables or it will make the diff a mess if I apply it everywhere.  It might be worth doing in a followup though.